configs/sst_kernel_rats-perf.yaml

```diff
@@ -12,6 +12,8 @@ data:
   - rteval-loads
   labels:
   - eln
+  limit_arches:
+    - x86_64
```

Comment:
This won't work. If you want to make all the package s above to only be on x86_64, you can move them under:

```
 arch_packages:
     x86_64:
       - tuned-profiles-realtime
       ...
```

and leave the packages field empty:

```
packages: []
```

The `limit_arches` is only available for `package_placeholders`.

See the [workload spec](https://github.com/minimization/content-resolver/blob/master/config_specs/workload.yaml#L61) for all available fields.